src/components/Proposals/ProposalCard.js

```diff
@@ -1,23 +1,83 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, GU } from '@aragon/ui'
+import { Card, GU, textStyle, useTheme } from '@aragon/ui'
+import { DISPUTABLE_VOTE_STATUSES } from './disputable-vote-statuses'
+import ProposalOption from './ProposalOption'
+import DisputableStatusLabel from './DisputableStatusLabel'
 
 function ProposalCard({ vote, onProposalClick }) {
-  const { voteId, context } = vote
+  const theme = useTheme()
+  const { context, voteId } = vote
+
   return (
     <Card
       onClick={() => onProposalClick(voteId)}
       css={`
+        display: grid;
+        grid-template-columns: 100%;
+        grid-template-rows: 1fr auto auto;
+        grid-gap: ${1 * GU}px;
         padding: ${3 * GU}px;
       `}
     >
-      <p>{context}</p>
+      <div
+        css={`
+          // overflow-wrap:anywhere and hyphens:auto are not supported yet by
+          // the latest versions of Safari (as of June 2020), which
+          // is why word-break:break-word has been added here.
+          hyphens: auto;
+          overflow-wrap: anywhere;
+          word-break: break-word;
+        `}
+      >
+        <span css="font-weight: bold">#{voteId}: </span>
+        <span>{context || 'No description provided'}</span>
+      </div>
```

Comment:
Should we make this a paragraph with `<strong>` for bolding? I also think we have room to make this description larger, in the designs it looks like 18px or `textStyle("body1")`

Reply:
Yes :) I left the `css="font-wight: bold` tough, cause otherwise aragonUI overrides the strong and b tag with font-weight: 600

Reply:
Is font-weight: 600 not bold enough? Perhaps we should consider a global style for `strong` inside a paragraph? I can see us needing to use it a lot.